Backend/routes/customerOrderRoutes.js

```
@@ -21,109 +25,119 @@ router.get("/", async (req, res) => {
+
+    try {
+      const order = await CustomerOrder.findById(req.params.id);
+      if (!order) return res.status(404).json({ error: "Order not found" });
+      if (order.status === "Cancelled") return res.status(400).json({ error: "Order already cancelled" });
+
+      order.status = "Cancelled";
+      await order.save();
+
+      const retailerOrder = await RetailerOrder.findOne({ shopId: order.shopId });
+      if (retailerOrder) {
+        const orderTotal = parseFloat(order.total.replace("$", ""));
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)